### PR TITLE
Fix the error in the password field in user struct

### PR DESCRIPTION
**Pull request by danielmkn**
Add PR links to the CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,16 +2,19 @@
...
+* Documentation improved for `project` resource to include limitations in Note section. 
+[GH-32](https://github.com/jfrog/terraform-provider-project/pull/32)
 
 ## 1.0.4 (Mar 22, 2022)
 
 BUG FIXES:
 
-* Project Key validation fixed as per the latest version. [GH-25]
+* Project Key validation fixed as per the latest version. 
+[GH-25](https://github.com/jfrog/terraform-provider-project/pull/25)
```
Comment (chb0github) [inline CHANGELOG.md:13]: We're doing issue and PR right

File: pkg/project/util_test.go [hunk 2]
```diff
@@ -158,7 +158,7 @@ func createTestUser(t *testing.T, name string, email string) {
 
 	type ArtifactoryUser struct {
 		Email    string `json:"email"`
-		Password string `json:"Password1!"`
+		Password string `json:"password"`
```
Comment (chb0github) [inline pkg/project/util_test.go:161]: I wondered about this

**Review by alexhung — APPROVED** [review]

**Review by chb0github — COMMENTED** [review]
.

File: pkg/project/util_test.go (hunk 2 above)
Comment (danielmkn) [reply pkg/project/util_test.go:161]: yep, you were right...

File: CHANGELOG.md (hunk 1 above)
Comment (danielmkn) [reply CHANGELOG.md]: added

Comment (chb0github) [reply CHANGELOG.md]: I thought `GH-25` meant issue 25? I guess.

**Review by chb0github — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (danielmkn) [reply CHANGELOG.md]: In AWS example, they mark issues with `#`

Comment (alexhung) [reply CHANGELOG.md]: https://www.terraform.io/plugin/sdkv2/best-practices/versioning#entry-format

> Each bullet also references the corresponding pull request number that contained the code changes, in the format of [GH-####]

**Pull request merged**